lib/matplotlib/animation.py

```diff
@@ -1393,25 +1392,19 @@ class TimedAnimation(Animation):
     Parameters
     ----------
     fig : `~matplotlib.figure.Figure`
-       The figure object that is used to get draw, resize, and any
-       other needed events.
-
-    interval : number, optional
-       Delay between frames in milliseconds.  Defaults to 200.
-
-    repeat_delay : number, optional
-        If the animation in repeated, adds a delay in milliseconds
-        before repeating the animation.  Defaults to ``None``.
-
-    repeat : bool, optional
-        Controls whether the animation should repeat when the sequence
-        of frames is completed.  Defaults to ``True``.
-
-    blit : bool, optional
-        Controls whether blitting is used to optimize drawing.  Defaults
-        to ``False``.
-
+        The figure object that is used to get draw, resize, and any
```

Comment:
Nitpick. That sentence is awkward. How about:

- `The figure object used to get needed events, such as draw or resize.`

(I'm still unhappy about the word `get`)